src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -903,9 +914,38 @@ private static bool HasImplicitNullableConversion(TypeSymbol source, TypeSymbol
                 return true;
             }
 
+            if (HasImplicitTupleConversion(unwrappedSource, unwrappedDestination, ref useSiteDiagnostics))
+            {
+                return true;
+            }
+
             return false;
         }
 
+        private bool HasImplicitTupleConversion(TypeSymbol source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            ImmutableArray<TypeSymbol> sourceTypes;
+            ImmutableArray<TypeSymbol> destTypes;
+
+            if (!source.TryGetElementTypesIfTupleOrCompatible(out sourceTypes) ||
```

Comment:
public Conversion CSharpCompilation.ClassifyConversion(ITypeSymbol source, ITypeSymbol destination)
public static Conversion CSharpExtensions.ClassifyConversion(this SemanticModel semanticModel, ExpressionSyntax expression, ITypeSymbol destination, bool isExplicitInSource = false)

---
In reply to: [66189168](https://github.com/dotnet/roslyn/pull/11681#discussion_r66189168) [](ancestors = 66189168)